mailbox/event/event-rabbitmq/src/test/java/org/apache/james/mailbox/events/RabbitMQEventBusTest.java

```diff
@@ -337,7 +342,7 @@ void dispatchShouldWorkAfterNetworkIssuesForOldRegistration() throws Exception {
                 rabbitMQExtension.getRabbitMQ().unpause();
 
                 eventBus.dispatch(EVENT, NO_KEYS).block();
-                verify(listener, after(THIRTY_SECONDS.toMillis()).times(1)).event(EVENT);
```

Comment:
when I wrote that verification, I want to verify that the listener will receive only one event all the time, but may be I was too strictly

Reply:
Yes that's what i tought, i think that it's enough to check that we receive the event